### DISCUSSION TITLE
german language

### DISCUSSION THREAD

**Discussion by Opferlamm** [fedorovvl/tso_client#136] [Ideas]
Hi Sirris, hi MkFera,
please can you add german language in this very nice program/script?
---------------------------------------------------------------
var _exudDepositVieweLang = {
	'de-de': {
		'TimeDepEnd': 'Zyklus',
		'BuffEnd': 'Buff Ende',
		'Buff': 'Buff',
		"Note": "*Die Zeiten bis zum Abbau der Vorkommen sind Schätzwerte und können, je nachdem wie weit die Entfernung zum Lager ist, variieren. Bitte zuerst F2 drücken! "
		},
-----------------------------------------------------------
"LAB", "Trait_LovelyGeologist" -->wrong translation  (Vorkommen)
------------------------------------------------------------
var _exudFriendsLang = {
	'de-de': {
		'online': 'Online',
		'since': 'Freun seit ',
		'term': 'Länge'
	},
-----------------------------------------------------------
const _exudGeneralsLang = {
	"de-de": {
		"ByName": "Name sortieren",
		"ByType": "Type sortieren",
		"ShowGuest": "Gäste zeigen",
		"HideGuest": "Gäste verbergen",
		"ShowUnselected": "Unmarkierte zeigen",
		"HideUnselected": "Unmarkierte verbergen",
		"SelectAll": "Alles auswählen",
		"Load": "Laden",
		"CommandSent": "Befehl gesendet",
		"ColumnOwner": "Besitzer",
		"IsGuest": " (* Gast) ",
		"SelectedFirst": "erst auswählen",
		"ExcludeStarMenu": "Sternemenü verbergen",
		"SkillTip": "Klicke auf den General um den Skill zu sehen."
	    },
-----------------------------------------------------------
const _exudspecDutyLang = { 
	"de-de": {
		"menuItemName": "Reisezeit"  ,
		"menuTitle": "Einsatzzeit"  ,
		"ColumnEstimated": "Geschätzt"  ,
		"ColumnArrival": "Ankunft"  ,
		"NoData": "Nichts gefunden!"  ,
		"YOU": "Ich",
		"HideShowGuest" : "Gast ein/aus"
	},
--------------------------------------------------
thanks a lot, best regards

**Comment by Opferlamm** [comment]
var _exudFriendsLang = {
'de-de': {
'online': 'Online',
'since': 'Freun_d_ seit ',
'term': 'Länge'
},

sry letterloss

**Comment by fedorovvl** [comment]
done

**Comment by fedorovvl** [comment]
done, thx for translation

**Reply by Opferlamm** [reply]
I have to thank, many German settlers players will be happy

**Comment by Opferlamm** [comment]
my keyboard is out of order   :-(

var exudFriendsLang = {
'de-de': {
'online': 'Online',
'since': 'Freund seit ',                           <------ this was wrong
'term': 'Länge'
},

--------------------------------------------------------
now in "german client" it looks so:

https://prnt.sc/LD3uWO6xn-_J

"Bezaubernde Vorkommen" is a very strange translation,
 without "Bezaubernde" it is ok.

---------------------------------------------------------
and last question
is it possible to create a checkbox to change the menu bar? 
then the user can decide that all is im tool or in menu bat
eg:
https://prnt.sc/ge1XtJuoBVUI


best regards and thx for your help so much
